lib/cocoapods/sources_manager.rb

```diff
@@ -80,8 +80,7 @@ def cdn_url?(url)
         credentials = netrc_info[netrc_host]
         uri_options[:http_basic_authentication] = credentials if credentials
 
-        response = OpenURI.open_uri(URI.join(url + '/', 'CocoaPods-version.yml'), uri_options)
-
+        response = OpenURI.open_uri(URI.join(url.chomp('/') + '/', 'CocoaPods-version.yml'), uri_options)
```

Comment:
`URI.join(url.chomp('/') + '/', 'CocoaPods-version.yml')` why not just `url.chomp('/') + '/', 'CocoaPods-version.yml'`?

Reply:
@jasl you're right, using `URI.join` seems redundant and cumbersome.

Reply:
changed